### PR TITLE
docs part 3

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
//cc @DevExpress/testcafe-docs

### CODE REVIEW

**Comment by MargaritaLoseva** [comment]
@DevExpress/testcafe-docs Should we add links to the features to the Overview topic?

**Comment by MargaritaLoseva** [comment]
closes #13

File: documentation/installing-testcafe-studio/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+* **MacOS**. Download the MacOS installation file and double-click it to expand its contents. Drag the TestCafe Studio application to the **Applications** folder, making it available in the **Launchpad**.
+
+    IMAGE
+
+* **Windows**. Download the Windows installation file and run it. The installation wizard will start and display the license agreement. If you accept the terms of the agreement, click **I Agree**. On the next page, specify if the software will be available to all users or just yourself and click **Install**. TestCafe Studio will be installed to *Program Files\DevExpress\TestCafe Studio*.
+
+    ![Installing TestCafe Studio on Windows](../../Images/installation/setup-windows.png)
+
+    After installation is completed, the instaler will create the TestCafe Studio shortcut on the desktop.
```
Comment (kirovboris) [inline documentation/installing-testcafe-studio/README.md]: It should add `TestCafe Studio` item in start menu as well.

File: documentation/overview/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+* To identify page elements, TestCafe Studio uses selectors. You won’t need to write the selectors yourself, they’re generated during the test recording. For each page element, TestCafe Studio produces a set of selectors, so you can pick the one that fits you best.
+
+* To check if the tested website works as expected, TestCafe Studio provides a set of assertions which allows you to perform different verifications.
+
+* Created tests can be run in all popular browsers: desktop and mobile, the ones with UI or headless.
+
+* TestCafe Studio has a built-in waiting mechanism that automatically checks that page elements are loaded during the test run. Thus, you don’t need to manually add timeouts. This helps you save time and get stable tests.
+
+* Comprehensive reports.
```
Comment (VasilyStrelyaev) [inline documentation/overview/README.md]: Maybe we should write a complete sentence here for consistency.

```
Once tests are finished, TestCafe Studio produces comprehensive reports that show...
```

**Comment by MargaritaLoseva** [comment]
@dirk-pieterse Please, review _installing-testcafe-studio/README.md_ and _overview/README.md_

File: documentation/overview/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+* TestCafe Studio provides a set of assertions which allows you to perform different verifications.
+
+* Created tests can be run in all desktop and mobile browsers (with or without a UI). See [Run Configurations](working-with-testcafe-studio/running-tests.md#run-configurations).
+
+* TestCafe Studio has a built-in waiting mechanism that automatically checks that page elements are loaded during the test run. You do not need to manually add timeouts. This helps you save time and get stable tests.
+
+* After the tests finish, TestCafe Studio provides a report with overall results and details for each test. See [Reports](../working-with-testcafe-studio/reports.md).
+
+* TestCafe Studio is made for users of all skill levels. The visual recorder allows creating tests without expert knowledge of JavaScript. For those who like to code, TestCafe Studio provides a powerful Code Editor with syntax highlight, code completion, parameter hints and so on. You can write code from scratch, or convert recorded tests to JavaScript code to edit them later. See [Writing Test Code](working-with-testcafe-studio/writing-test-code.md).
```
Comment (dirk-pieterse) [inline documentation/overview/README.md]: Rewrite.

File: documentation/overview/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+layout: docs
+title: Overview
+permalink: /documentation/overview/
+---
+# Overview
+
+TestCafe Studio is a cross-platform IDE for end-to-end web testing that does not require WebDriver or other tools. It works on Windows, MacOS, and Linux, and in every popular browser. You can install it in a couple of minutes and start testing right away.
+
+TestCafe Studio provides advanced features that help you to test web apps fast.
```
Comment (dirk-pieterse) [inline documentation/overview/README.md]: Rewrite.

File: documentation/overview/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+* You can create, edit and maintain end-to-end tests with a visual recorder without writing code. See [Recording Tests](../working-with-testcafe-studio/recording-tests.md).
+
+* TestCafe Studio uses selectors which are generated during the test recording to identify page elements. TestCafe Studio produces a set of selectors for each page element, so you can pick the one that fits you best.
+
+* TestCafe Studio provides a set of assertions which allows you to perform different verifications.
+
+* Created tests can be run in all desktop and mobile browsers (with or without a UI). See [Run Configurations](working-with-testcafe-studio/running-tests.md#run-configurations).
+
+* TestCafe Studio has a built-in waiting mechanism that automatically checks that page elements are loaded during the test run. You do not need to manually add timeouts. This helps you save time and get stable tests.
```
Comment (dirk-pieterse) [inline documentation/overview/README.md]: Rewrite.

**Pull request merged**